File: assign_groups/assign_groups.go
# https://codefights.com/challenge/ogcRDybjWMDEbph8W

import "sort"

func assignGroups(timeEstimates []int, groupSize int) int {
    timeTotal := 0  // total time estimate
    sort.Sort(sort.Reverse(sort.IntSlice(timeEstimates)))
    
    for i := 0; i < len(timeEstimates); i += groupSize {
        timeTotal += timeEstimates[i]
    }
    
    return timeTotal
}

File: assign_groups/assign_groups.rb
# https://codefights.com/challenge/ogcRDybjWMDEbph8W

def assignGroups(timeEstimates, groupSize)
    n = timeEstimates.size
    times = timeEstimates.sort.reverse
    0.step(n-1, groupSize).sum { |x| times[x] }
end
